docs/project/set-up-dev-env.md

```diff
@@ -255,25 +255,25 @@ example, you'll edit the help for the `attach` subcommand.
     Your location should be different because, at least, your username is
     different.
 
-3. Open the `api/client/attach.go` file.
+3. Open the `api/client/container/attach.go` file.
 
 4. Edit the command's help message.
 
     For example, you can edit this line:
 
-    ```go
-    noStdin := cmd.Bool([]string{"-no-stdin"}, false, "Do not attach STDIN")
-    ```
+	```go
+	flags.BoolVar(&opts.noStdin, "no-stdin", false, "Do not attach STDIN")
+	```
```

Comment:
I think your editor is configured to use `tabs` for indenting. We standardized on using spaces for that (in our `Markdown` files). Can you update your changes to use spaces instead?